R/features.R

```diff
@@ -54,15 +52,17 @@ feature_combinations <- function(m, exact = TRUE, n_combinations = 200,
     stop("Currently we are not supporting cases where the number of features is greater than 30.")
   }
 
-  if (!exact && n_combinations > (2^m - 2) && !reduce_dim) {
+  if (!exact && n_combinations > (2^m - 2)) {
     n_combinations <- 2^m - 2
+    exact <- TRUE
     cat(sprintf("n_combinations is larger than or equal to 2^m = %d. Using exact instead.", 2^m))
   }
 
   if (exact) {
     dt <- feature_exact(m, weight_zero_m)
   } else {
-    dt <- feature_not_exact(m, n_combinations, weight_zero_m, reduce_dim)
+    dt <- feature_not_exact(m, n_combinations, weight_zero_m)
+    dt[, p := NULL]
```

Comment:
Is there a reason you did not remove this within feature_not_exact?

Reply:
I didn't want to remove it inside the function, since I want to do unit tests for the probabilities we're using. This is the same as we did before (when the Shalpey values was equal to the probabilities).